csrc/preprocess/transform/pad.cpp

```diff
@@ -22,7 +22,17 @@ PadImpl::PadImpl(const Value& args) : TransformImpl(args) {
   }
 
   arg_.size_divisor = args.value("size_divisor", 1);
-  arg_.pad_val = args.value("pad_val", 0.0f);
+  if (args.contains("pad_val")) {
```

Comment:
@AllentDan, please check whether the Pad transform goes with its spec in `https://openmmlab.feishu.cn/docs/doccn0yydblfiDj0FNTQgkkTmEC#4UvzZT`

Reply:
Seems `pad_val (float, optional) - Padding value, 0 by default.` in the document.